ledger-core/virtual/integration/method_test.go

```diff
@@ -1779,7 +1814,7 @@ func TestVirtual_Method_IntolerableCallChangeState(t *testing.T) {
 		p1        = server.GetPulse().PulseNumber
 		isolation = contract.MethodIsolation{
 			Interference: contract.CallIntolerable,
-			State:        contract.CallValidated,
+			State:        contract.CallDirty,
```

Comment:
Why don't you add a ValidatedState to the object? According to documentation, we have intolerable+validated, tolerable+dirty calls in R0.

Reply:
done